dbt_airflow_factory/tasks_builder/builder.py

```diff
@@ -18,6 +19,35 @@
 from dbt_airflow_factory.tasks_builder.graph import DbtAirflowGraph
 
 
+@dataclass
+class ResultTaskData:
+    nodes_data: ModelExecutionTask
+    schema: str
+    model_name: str
```

Comment:
If you include schema and model_name into ModelExecutionTask it will the code will be simpler. Why do you want to wrapper?